From 749594928cc1a0e452c4c791fcbea2f8f4a7e259 Mon Sep 17 00:00:00 2001
From: Mike Westerhof <mwester@dls.net>
Date: Fri, 8 Aug 2008 13:08:03 +0100
Subject: [PATCH] commit 830ea3d0c27c0c750b7bf1b56c002ee7943f3edc

    gta0x-log-serial-rx-error.patch

    This patch causes a KERN_DEBUG message to be printed each time an
    error status is read from a UART.  This is intended to facilitate
    the reporting of more useful problem and bug reports from users
    in the field.

    Signed-off-by: Mike Westerhof <mwester@dls.net>
---
 drivers/serial/s3c2410.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/serial/s3c2410.c b/drivers/serial/s3c2410.c
index fef3c8f..5652905 100644
--- a/drivers/serial/s3c2410.c
+++ b/drivers/serial/s3c2410.c
@@ -357,7 +357,7 @@ s3c24xx_serial_rx_chars(int irq, void *dev_id)
 		port->icount.rx++;
 
 		if (unlikely(uerstat & S3C2410_UERSTAT_ANY)) {
-			dbg("rxerr: port ch=0x%02x, rxs=0x%08x\n",
+			printk(KERN_DEBUG "rxerr: port ch=0x%02x, rxs=0x%08x\n",
 			    ch, uerstat);
 
 			/* check for break */
-- 
1.5.6.5

